php/send-http-request/index.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+require 'vendor/autoload.php';
+
+use GuzzleHttp\Client;
+
+return function($req, $res) {
+    try {
+        $payload = \json_decode($req['payload'], true);
+        $url = \trim($payload['url']);
+        $method = \trim($payload['method']);
+        $body = \trim($payload['body']);
+        $headers = $payload['headers'];
+    } catch(\Exception $err) {
+        \var_dump($err);
+        throw new \Exception('Payload is invalid.');
```

Comment:
Let's return a formatted response instead of throwing an exception. Something like:

```suggestion
        $res->json([
            'success' => false,
            'message' => 'Payload is invalid: ' . $err->getMessage()
        ]);
        return;
```